src/components/container/SurveyTaskList/SurveyTaskList.tsx

```diff
@@ -15,6 +15,7 @@ export interface SurveyTaskListProps {
 	sequential?: boolean,
 	title?: string,
 	surveys?: string[],
+	category?: string,
```

Comment:
Should we consider multiple categories?

Reply:
Yeah, I did consider that at some point and wasn't sure. But @ajmenca suggested we move the filtering to the survey task query params instead (see my new commit), and it doesn't seem like the [delegated API](https://github.com/CareEvolution/Consumers/blob/33226d9ab43356445b0ea9ef4c246cbbb2668f56/Mogul/Code/ResearchKit/Api/External/V1/DelegatedParticipant/DelegatedSurveyTasksController.cs#L64-L65) supports multiple categories (unlike with survey names).